src/charts/UsageByParticipantTier.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+import { Line } from 'react-chartjs-2';
+import { get } from 'lodash';
+import moment from 'moment';
+import Loading from '../components/Loading';
+import ErrorMessage from '../components/ErrorMessage';
+
+const apiKey = process.env.REACT_APP_API_KEY;
+
+const query = gql`
+  {
+    project(projectId: ${apiKey}) {
+      projectData(
+        start: ${moment().subtract(10, 'days')},
+        interval: DAILY
+      ) {
+        resources {
+          intervalStart,
+          intervalEnd,
+          usage {
+          participantMinutes{
+            from1To2Publishers
+            from3To6Publishers
+            from7To8Publishers
+            from1To4Publishers
+            from5To8Publishers
+            from1To8Publishers
+            from1To10Publishers
+            from9To10Publishers
+            from11To35Publishers
+            from11To20Publishers
+            from11To35Publishers
+            from36PlusPublishers
+            from21To35Publishers
+            from36To40Publishers
+            from41PlusPublishers
+          }
```

Comment:
This should be indented.

```suggestion
            participantMinutes{
              from1To2Publishers
              from3To6Publishers
              from7To8Publishers
              from1To4Publishers
              from5To8Publishers
              from1To8Publishers
              from1To10Publishers
              from9To10Publishers
              from11To35Publishers
              from11To20Publishers
              from11To35Publishers
              from36PlusPublishers
              from21To35Publishers
              from36To40Publishers
              from41PlusPublishers
            }
```